tests/Unit/Evolution/Systems/Cce/Test_OptionTags.cpp

```diff
@@ -0,0 +1,55 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "Evolution/Systems/Cce/OptionTags.hpp"
+#include "Evolution/Systems/Cce/ReadBoundaryDataH5.hpp"
+#include "NumericalAlgorithms/Interpolation/CubicSpanInterpolator.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp"
+#include "Utilities/FileSystem.hpp"
+#include "tests/Unit/Evolution/Systems/Cce/BoundaryTestHelpers.hpp"
+#include "tests/Unit/TestCreation.hpp"
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.Cce.OptionTags",
+                  "[Unit][Evolution]") {
+  CHECK(TestHelpers::test_creation<size_t, Cce::OptionTags::LMax>("8") == 8u);
```

Comment:
in `Utilities/Literals.hpp` there is code to make `8u` -> `8_st` for `size_t`, please use that